Makefile.am

```diff
@@ -181,6 +193,9 @@ rpm: dist
 		--define="vdsm_release $(PACKAGE_RELEASE)" \
 		$(DIST_ARCHIVES)
 
+clean-rpm:
+	rm -rf $(RPM_DIR)
```

Comment:
This delete the entire rpbbuild, which may be bad if you tried to build other packages
and you want to use the built packages. I do this when building vdsm since I never
build other stuff in rpmbuild, but I do the explicit "rm -rf ~/rpmbuild", so I'm not
surprised by the results.

I think we should start using our own rpmbuild directory - using a clean directory per
build, used only for vdsm. See imageio makefile for example:
https://github.com/oVirt/ovirt-imageio/blob/0568314462a47e8c3ddffc4c29520b67ced41c33/Makefile#L31